spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/model/Owner.java

```diff
@@ -61,32 +61,26 @@ public class Owner {
     @Getter
     @Setter
     @Column(name = "first_name")
-    @NotBlank
```

Comment:
@arey from the given link ,its not possible to apply records to the entity class for the reasons specified in the article.  https://vladmihalcea.com/java-records-jpa-hibernate/ 


Hence I decoupled and introduced DTO  record class OwnerRequest and DTO validation are extracted to the DTO record class. since the validation is happening at the first level at DTO request , I removed the validation from the entity class.

Reply:
This may be duplication, but I think we could keep Bean Validation annotations on the JPA Entities. If we use the `Owner` class without the `OwnerRequest` we can be sure that valid data will be inserted into the database

Reply:
done